sql/core/src/test/scala/org/apache/spark/sql/DataFrameAggregateSuite.scala

```diff
@@ -1012,4 +1012,20 @@ class DataFrameAggregateSuite extends QueryTest
       }
     }
   }
+
+  test("SPARK-32038: NormalizeFloatingNumbers should work on distinct aggregate") {
+    withTempView("view") {
+      val POS_NAN_1 = java.lang.Float.intBitsToFloat(0x7f800001)
+      val POS_NAN_2 = java.lang.Float.intBitsToFloat(0x7fffffff)
```

Comment:
nit: any reason to use uppercases for the variables?

Reply:
No, just copied from the original report on the jira.

Reply:
Let's rename it. It's actually a bit odds to have a constant-like name

Reply:
hahaha, I see. Yea, I think its better to rename it, too.

Reply:
ok. :)